pkg/broker/statestorage/statestorage.go

```diff
@@ -44,16 +44,16 @@ type RealmStateStorage struct {
 	Logger       *zap.SugaredLogger
 }
 
-func client(baseURL string, k credentials.APIKey) (*mongodbatlas.Client, error) {
+func client(baseURL string, userAgent string, k credentials.APIKey) (*mongodbatlas.Client, error) {
 	hc, err := digest.NewTransport(k.PublicKey, k.PrivateKey).Client()
 	if err != nil {
 		return nil, errors.Wrap(err, "cannot create Digest client")
 	}
 
-	return mongodbatlas.New(hc, mongodbatlas.SetBaseURL(baseURL))
+	return mongodbatlas.New(hc, mongodbatlas.SetBaseURL(baseURL), mongodbatlas.SetUserAgent(userAgent))
 }
 
-func Get(ctx context.Context, key credentials.APIKey, atlasURL string, realmURL string, logger *zap.SugaredLogger) (*RealmStateStorage, error) {
+func Get(ctx context.Context, key credentials.APIKey, userAgent string, atlasURL string, realmURL string, logger *zap.SugaredLogger) (*RealmStateStorage, error) {
```

Comment:
[q] will the "userAgent" be different for different calls? Maybe it makes sense to encapsulate it into `client` method?

Reply:
`userAgent` is always set by `Broker`, which, in turn, gets configured by `main`

Reply:
But it's basically global and configured only once? Maybe having a global variable will save some method parameters?

Reply:
Several concerns with that:
1. That would mean creating a new package just for that, as this value is referenced from multiple existing packages (also I'm generally against them outside of `main.go` or tests)
1. AtlasURL & RealmURL could also become "global" following the same logic
1. I'm not sure why having an extra method parameter is bad - this is still less code than adding a new package for globals